python/torch_mlir/eager_mode/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+#-------------------------------------------------------------------------------
+# Setup PyTorch
+#-------------------------------------------------------------------------------
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules")
+TorchMLIRProbeForPyTorchInstall()
```

Comment:
do we still need to probe this even though it is pure Python?

Reply:
yea i guess not